agent/trend_watcher.py

```
@@ -641,6 +830,55 @@ def _dispatch_to_improvement_loop(self, alerts: list[TrendAlert]) -> None:
+                        "relevance_score": alert.relevance_score,
+                        "dispatched_at": time.time(),
+                    },
+                }
+                with httpx.Client() as client:
+                    client.post(
+                        f"{hermes_url.rstrip('/')}/tasks",
+                        json=payload,
+                        timeout=10,
+                    )
+                log.info("TrendWatcher: dispatched alert %s to Hermes", alert.title[:50])
+            except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)